bug-1676470-pref-warpbuilder-experiment-release-83-83.toml

```diff
@@ -0,0 +1,369 @@
+[experiment]
+segments = [
+    'low_cpu', 'med_cpu', 'high_cpu',
+    'low_mem', 'med_mem', 'high_mem',
+    'regular_users_v3', 'new_or_resurrected_v3'
+]
+
+[metrics]
+overall = [
+    'perf_page_load_time_ms', 'perf_page_load_time_from_responsestart_ms',
+    'perf_dom_content_loaded_time_ms', 'perf_dom_content_loaded_time_from_responsestart_ms',
+    'perf_first_contentful_paint_ms', 'perf_first_contentful_paint_from_responsestart_ms',
+    'perf_request_animation_callback_pageload_ms', 'perf_request_animation_callback_non_pageload_ms',
+    'js_execution_proportion',
+    'time_to_first_interaction_ms', 'input_event_response_ms',
+    'content_keypress_present_latency', 'gpu_keypress_present_latency',
+    'load_input_event_response_ms', 'load_input_event_response_no_preload_ms', 'load_input_event_response_preload_ms',
+    'about_home_topsites_first_paint',
+    'fx_new_window_ms'
+]
+    [metrics.perf_page_load_time_ms]
+    select_expression = '{{agg_histogram_mean("payload.processes.content.histograms.perf_page_load_time_ms")}}'
+    data_source = 'main'
+
+        [metrics.perf_page_load_time_ms.statistics.bootstrap_mean]
+        pre_treatments = ["remove_nulls"]
+
+        [metrics.perf_page_load_time_ms.statistics.deciles]
+        pre_treatments = ["remove_nulls"]
+
+        [metrics.perf_page_load_time_ms.statistics.kernel_density_estimate]
+        pre_treatments = ["remove_nulls"]
```

Comment:
It's probably best to specify log_space = true for all of the KDEs as well if you're expecting log-normal distributions (it's a relatively recent addition)

Reply:
added